go/kbfs/libkbfs/interfaces.go

```diff
@@ -1855,6 +1855,23 @@ type Observer interface {
 	TlfHandleChange(ctx context.Context, newHandle *tlfhandle.Handle)
 }
 
+// SyncedTlfObserver can be notified when a sync has started for a
+// synced TLF, or when a TLF becomes unsynced.  The notification
+// callbacks should not block, or make any calls to the Notifier
+// interface.
+type SyncedTlfObserver interface {
+	// FullSyncStarted announces that a new full sync has begun for
+	// the given tlf ID, and it will be completed (or canceled) once
```

Comment:
this phrasing is confusing to me. Do you mean that the caller will close the channel after the sync is done/cancelled?

Reply:
Yes good catch, will fix.